assets/src/components/notificationCard.tsx

```diff
@@ -165,6 +180,24 @@ const description = (
         routeAtCreation
       )
     }
+
+    case NotificationType.Detour: {
+      return (
+        <>
+          <div className="d-flex flex-row gap-2">
+            <RoutePill routeName={notification.content.route} />
```

Comment:
(Very non-blocking): This combination of route-pill-plus-headsign-etc seems to be happening everywhere in the detours world. Would it be worth pulling it out to its own re-usable component sometime soon?

Reply:
I personally don't think it's worth it _quite_ yet?